src/docs/resources/platform-adaptations.md

```diff
@@ -0,0 +1,218 @@
+---
+title: Platform Specific Behaviors and Adaptations
+---
+
+## Adaptation Philosophy
+
+There are generally 3 categories of platform adaptiveness:
+
+1. Things that are behaviors of the OS environment (such as text editing and
+scrolling) and that would be 'wrong' a different behavior took place.
+2. Things that are conventionally implemented in apps using the OEM's SDKs
+(such as using parallel tabs on iOS or showing an [android.app.AlertDialog](https://developer.android.com/reference/android/app/AlertDialog.html)
+on Android).
+3. Correct usage of corresponding platform APIs to export to TalkBack/VoiceOver,
+status bar color theme etc. and to import notch metrics, locale settings etc.
```

Comment:
Ack. This sentence has TWO "etc."s in it. It shouldn't have any, but definitely not two!  Also, add a comma after "status bar". Currently, this sentence is hard to parse.

Reply:
This part is actually kinda in the gray area. I just removed 3